src/debugger/PerFileBreakpointMap.js

```diff
@@ -53,22 +54,28 @@ export class PerFileBreakpointMap {
 
   removeBreakpoint(line: number, column: number = 0) {
     let key = this._getKey(line, column);
-    if (key in this.breakpoints) {
-      delete this.breakpoints[key];
+    if (key in this._breakpoints) {
+      this._breakpoints.delete(key);
     }
   }
 
   enableBreakpoint(line: number, column: number = 0) {
     let key = this._getKey(line, column);
-    if (key in this.breakpoints) {
-      this.breakpoints[key].enabled = true;
+    if (this._breakpoints.has(key)) {
+      let breakpoint = this._breakpoints.get(key);
+      // flow complains without this invariant
```

Comment:
Probably because it does not assume that the if condition guarantees that breakpoint is not undefined. If you just call get, this problem goes away.